command/flags.go

```diff
@@ -323,6 +323,10 @@ func noDockerCapabilitiesFlag(f *flag.FlagSet) *bool {
 	return f.Bool("no-docker-capabilities", false, "Disable all docker capabilities")
 }
 
+func dockerDevicesFlag(f *flag.FlagSet) *string {
+	return f.String("docker-devices", "", "The list of device mapping if any, separated with a comma. format: src:opt:dst")
```

Comment:
This is incoherent with [https://github.com/squarescale/front/commit/25aff16727166a359d813a70703ab0d3d746c67e](https://github.com/squarescale/front/commit/25aff16727166a359d813a70703ab0d3d746c67e)

must be something like 
```
"The list of device mappings if any, separated with commas. format: src:dst[:opt][,src:dst[:opt]]*"
```